jenkinsTerraform/jenkins-agent01-installs.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+#################### Install Java ####################
+sudo apt-get update -y
+apt-get install -y openjdk-8-jre -y
+sudo apt-get update -y
+
+#################### Set up Docker's apt repository ####################
+
+#################### Add Docker's official GPG key ####################
+sudo apt-get update
+sudo apt-get install ca-certificates curl gnupg
+sudo install -m 0755 -d /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+sudo chmod a+r /etc/apt/keyrings/docker.gpg
+
+#################### Add the repository to Apt sources ####################
+echo \
+  "deb [arch="$(dpkg --print-architecture)" signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  "$(. /etc/os-release && echo "$VERSION_CODENAME")" stable" | \
+  sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt-get update
+
+#################### Install the Docker packages  ####################
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+
+#################### Verify that the Docker Engine installation is successful by running the hello-world image ####################
+sudo docker run hello-world
+
+#################### Done! ####################
+echo "Installations successfully installed!"
+
+exit 0
```

Comment:
```suggestion
#App dependencies
sudo apt update
sudo apt install -y software-properties-common 
sudo add-apt-repository -y ppa:deadsnakes/ppa 
sudo apt install -y python3.7 
sudo apt install -y python3.7-venv
sudo apt install -y build-essential 
sudo apt install -y libmysqlclient-dev
sudo apt install -y python3.7-dev
exit 0
```

This agent will be testing the backend of the app so the application dependencies need to be downloaded on this agent